Branch-TestBed-Swift/TestBed-Swift/AppDelegate.swift

```diff
@@ -36,35 +45,28 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
                 DataStore.setActivePendingSetDebugEnabled(false)
             }
             
-            let navigationController = UIStoryboard(name: "Main", bundle: nil).instantiateInitialViewController() as! UINavigationController
-            branch.registerDeepLinkController(navigationController, forKey:"~referring_link")
-            
             // Required. Initialize session. automaticallyDisplayDeepLinkController is optional (default is false).
-            branch.initSession(launchOptions: launchOptions, automaticallyDisplayDeepLinkController: true, deepLinkHandler: { params, error in
+            branch.initSession(launchOptions: launchOptions, automaticallyDisplayDeepLinkController: false, deepLinkHandler: { params, error in
```

Comment:
let's change to use a the different initSession method that doesn't have `automaticallyDisplayDeepLinkController` in the arguments. Can you remove it altogether?